### PR TITLE
Sql

### PR DESCRIPTION
**Pull request by TRACYMUSIKER**
- added FE/src/component - SignupScreen & SignupForm 
-successfully allows a user to sign in (/#signup) and saves email/pw in SQL DB
-SQL DB TABLE  and other .sql files:
    - table for user
    - dummy data
    - example queries
-db.js (functions for strictly SQL queries)
plz review.. :)
@smuntean @hglasser

### CODE REVIEW

File: Backend/server.js [hunk 1]
```diff
@@ -11,11 +12,22 @@ let allowCORS = (req, res, next) => {
...
+}
+
+
 app.use(bodyParser.json());
 
 app.use(express.static("../Frontend/build"));
 app.use(allowCORS);
 app.use(publicRouter);
+// app.post('/authenticate', authenticate);
+app.post('/signup', signup);
```
Comment (sarathrasher) [inline Backend/server.js]: This post request should be to publicRouter, not app

**Comment by TRACYMUSIKER** [comment]
@smuntean @hglasser

**Review by hannah-rackley — APPROVED** [review]
So much work! This looks good

**Comment by TRACYMUSIKER** [comment]
@TRACYMUSIKER @hglasser please review and approve

**Pull request merged**